### ISSUE TITLE
Click in chat message container to focus message input

### ISSUE THREAD

**Issue by adambuczynski** [gitterHQ/gitter#1049]
I don't know if this is intentional behaviour or not, but every time when I press the `enter` key while the chat message box doesn't have focus, Gitter goes ahead and focusses on another room in my list. It's always the same room (last in the list), no matter which room I'm in.

I find it hard to believe that this behaviour is somehow intentional, as it's quite annoying to lose focus from your current chat room in that manner.

This is on Mac OS X 10.11.2 using the latest Gitter desktop app, version 1.162.

**Comment by MadLittleMods** [comment]
I'm curious why you are pushing enter when nothing has focus?

This happens in the web and desktop app.

For me, it always goes to the page loaded room or the room last switched to via `Cmd + Ctrl + Arrow keys`.

**Comment by adambuczynski** [comment]
Glad you asked. I do that because in all the other chat clients that I'm used to using (e.g. Google Hangouts, HipChat, ..), when you click on the body of the chat room (e.g. not a specific clickable element like a username, but just the text content area of the conversation), the chat input box receives focus.

So instead of having to click on a tiny area of the window in order to be able to start chatting, I can click quickly on a nearly 80% area of the chat window and then immediately start typing. 

Being used to that, I of course automatically tried to do that in Gitter as well, but apparently that doesn't work. So as a result, I start typing without realizing that the chat input box doesn't have focus and then once I am ready to send the message and hit enter, the app would focus on the other chat room.

So personally, I think the solution would be to focus the chat input box on any unhandled clicks on the chat room area, as this would be in line with other major chat applications, and improve accessibility.

**Comment by adambuczynski** [comment]
@MadLittleMods any thoughts on the above?

**Comment by MadLittleMods** [comment]
@adambuczynski It was labeled earlier as something worth tackling, not only to fix the odd behavior currently, but to also implement what you like about other platforms(clicking on chat area, to focus text input).

**Comment by adambuczynski** [comment]
Great, is pebble a label for an upcoming version/release?

**Comment by MadLittleMods** [comment]
@adambuczynski A pebble is just an estimation of it being a small task to solve this issue.

**Comment by adambuczynski** [comment]
Thanks for clarifying

**Comment by adambuczynski** [comment]
@MadLittleMods Any chance this fix could make it into the next Gitter release as well?

It's still driving me crazy on a day-to-day basis, as I'm also using HipChat for work and there clicking on the chat history focuses the text input, so out of habit I also keep doing it for the Gitter window :(

**Comment by MadLittleMods** [comment]
We are continually working on the accessibility issues and priority is based on our judgement of popularity and value. No guarantees this will make it in the next release but it is a good issue to tackle sometime soon.

**Comment by adambuczynski** [comment]
Cheers thanks

**Comment by liatreis** [comment]
+1

**Comment by adambuczynski** [comment]
Hello, any update on this issue? Still as frustrating as ever, and in the meantime HipChat is looking more and more polished and sleek...
